pages/common/katana.md

```diff
@@ -0,0 +1,28 @@
+# katana
+
+> A next-generation crawling and spidering tool.
```

Comment:
```suggestion
> A fast crawler focused on execution in automation pipelines offering both headless and non-headless crawling.
> See also: `gau`, `scrapy`, `waymore`.
```
- "web crawling" is a synonymous for "web spidering", right? I took this description from the usage section of their README
- I added a "See also" section for some similar tools

Reply:
My description was taken from the About section (top right) on the project's GitHub page. I am ok with your suggestion.